package.json

```diff
@@ -2,46 +2,50 @@
   "name": "gulp-htmlmin",
   "description": "gulp plugin to minify HTML.",
   "version": "1.3.0",
-  "author": {
-    "name": "Jon Schlinkert",
-    "url": "https://github.com/jonschlinkert"
-  },
+  "homepage": "https://github.com/jonschlinkert/gulp-htmlmin",
+  "author": "Jon Schlinkert (https://github.com/jonschlinkert)",
   "contributors": [
     "Shinnosuke Watanabe <snnskwtnb@gmail.com> (https://github.com/shinnn)"
   ],
   "repository": "jonschlinkert/gulp-htmlmin",
+  "bugs": {
+    "url": "https://github.com/jonschlinkert/expand-files/issues"
+  },
   "license": "MIT",
-  "keywords": [
-    "format",
-    "gulp",
-    "gulpplugin",
-    "htm",
-    "html",
-    "minifier",
-    "minification",
-    "minify"
-  ],
   "files": [
     "index.js"
   ],
+  "main": "index.js",
```

Comment:
> It's a bug of the linter.

Lol how is that a bug? I asked you to show me where you're getting the error, I'm getting the sense that was just an attempt at establishing authority on the matter. So let's start with where you're getting the error, then show me where it's an idiomatic best practice to not have `main` in package.json when the project actually has an index file as it's main entry point. It's a very trivial matter to remove that config option from my linter, but I'd like to have a reason. The point you made only applies to projects that list a file that doesn't exist. e.g. not this one.